README.md

```diff
@@ -63,6 +63,11 @@ Open id configuration
 | `keycloak.bearer-only` | `true` | See [Keycloak configuration](https://www.keycloak.org/docs/latest/securing_apps/#_java_adapter_config) |
 | `keycloak.disable-trust-manager` | `false` | See [Keycloak configuration](https://www.keycloak.org/docs/latest/securing_apps/#_java_adapter_config) |
 
+CORS configuration :
+| Key | Default | Description |
+| --------------------- | ------- | ------------------------------------------------------------------ |
+| `cors.origins` | `*` | See [CORS configuration with spring](https://spring.io/blog/2015/06/08/cors-support-in-spring-framework)|

```

Comment:
I think this configuration parameter should not redirect to spring's documentation. It should be as implementation-agnostic as possible as this is user's documentation.  (note that Keycloak's configuration points to the Keycloak dev documentation as we are directly using their configuration properties).  

We could point to https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS for example ? Not sure